packages/@expo/cli/src/start/server/MCP.ts

```diff
@@ -58,9 +63,15 @@ export async function maybeCreateMCPServerAsync(
     const server: McpServerProxy = new TunnelMcpServerProxy(mcpServerUrl, {
       logger,
       wsHeaders: createAuthHeaders(),
+      projectRoot,
+      devServerUrl,
     });
     addMcpCapabilities(server, projectRoot);
 
+    installExitHooks(() => {
+      server.close();
```

Comment:
Since `installExitHooks` allows for async functions maybe we should return the promise from close? Or just pass `close` as the parameter?